### DISCUSSION TITLE
关于v2ray的实现原理疑问

### DISCUSSION THREAD

**Discussion by moonloopy** [v2fly/v2ray-core#2480] [Q&A]
用clash的时候，用Clash配置系统代理后，在idea软件里的代理选项设置成“无代理”时，照样可以在idea里走代理。
![image](https://user-images.githubusercontent.com/45658884/234540503-533d2ac1-ff66-4e67-937f-acf9cbfe51ac.png)
而用v2ray的时候，配置代理后，如果在idea软件里设置成“无代理”时，则在idea软件里不可以走代理。这是为什么呢？

**Comment by moonloopy** [comment]
……没开tun模式

**Reply by mydogshitgold** [reply]
I think you've probably confused v2ray-core with third-party frontends. There is currently no built-in TUN for v2ray-core. You may get more relevant answers from your frontend's repo.
